modules/microshift-examples-of-usecase.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * microshift//running_applications/microshift-deleting-resource-manifests.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-examples-of-usecase_{context}"]
+= Use cases for resource manifest deletion
+
+The following explains the use case in which the resource manifest deletion is used.
+
+[id="data-removal-scenario_{context}"]
+== Data removal scenario
+Use the following procedure in the data removal scenario to completely delete the resource defined in the kustomize manifest file.
+
+.Procedure
+
+. Identify the file that needs to be placed in the `delete` subdirectories.
+. Move the manifest file into one of the `delete` subdirectories  under the configured manifests directory by running the following command:
++
+[source,terminal]
+----
+[sudo] mv /usr/lib/microshift/manifests.d/some-user-manifest /usr/lib/microshift/manifests.d/delete/some-user-manifest
+----
+. {microshift-short} detects and removes the resource after the manifest file is placed in the `delete` subdirectories.
```

Comment:
MicroShift needs to be restarted to make it effective
`sudo systemctl restart microshift`

Reply:
done